specs/lib/generateNormalsSpec.js

```diff
@@ -42,11 +42,11 @@ describe('generateNormals', function(){
         expect(gltf.meshes).toEqual(gltfCopy.meshes);
     });
 
-    it('generates hard normals if they do not exist', function() {
+    it('generates face normals if they do not exist', function() {
         var gltf = gltfNoNormals;
         var byteLengthBefore = 168;
         generateNormals(gltf, {
-            hardNormals : true
+            faceNormals : true
         });
```

Comment:
I clarified the wording in the other test, smooth normals is the default so it will be tested there.